src/middleware/validator.js

```diff
@@ -15,9 +15,16 @@ const performValidation = (ctx, schema) => {
     throw new Error(`Invalid request body`)
   }
 
+  const coerceTypes =
+    configurations.validation.coerceTypes === 'false'
+      ? false
+      : configurations.validation.coerceTypes
```

Comment:
@BMartinos, with this logic, if the `configurations.validation.coerceTypes` is **true** or anything other than the string **false**, the coerceTypes variable will be set to a value of type string. And also for consistency, this logic can be in the config file

Reply:
@bradsawadye correct, If left it this way intentially for now, but maybe theres a better way. The reason for this is that multiple types can be supplied. This isnt just a Boolean value, but also a string value. 
https://github.com/epoberezkin/ajv/blob/master/COERCION.md#ajv-type-coercion-rules
https://github.com/epoberezkin/ajv#coercing-data-types

This means the config option can be `true`, `false` or `'array'`

If the environment variable being supplied is a string `'false'`, this will equate it to an actual boolean `false`, otherwise anything else in the string will equate to `true`

Let me see if there isnt a better way to handle this

Reply:
Okay cool

Reply:
done in b95e138